package.json

```diff
@@ -63,8 +57,7 @@
     "rxjs": "^6.5.2"
   },
   "devDependencies": {
-    "husky": "^3.0.1",
-    "lint-staged": "^9.2.0",
```

Comment:
Why'd you strip out the lint hook stuff?

Reply:
Ah, sorry this was an accident! It doesn't work for me locally, so I had to remove it to get anything to build (my weird `execa` issue...) I will add it back.

Reply:
Yeah, that's what I assumed after you mentioned the trouble yesterday